messaging/src/main/java/org/axonframework/eventhandling/pooled/Coordinator.java

```diff
@@ -510,6 +537,32 @@ Builder maxClaimedSegments(int maxClaimedSegments) {
             return this;
         }
 
+        /**
+         * Sets the initial segment count used to create segments on start up.
```

Comment:
It would be nice to tell what the default `initialSegmentCount` is on the JavaDoc here! :-)

Reply:
added